buildSrc/src/main/rocker/com/uber/okbuck/template/config/OkbuckTargets.rocker.raw

```diff
@@ -187,7 +187,9 @@ def okbuck_lint(
     cmd += "--fullpath "
 
     cmd += "{} `dirname {}` >$LOGFILE 2>/dev/null; ".format(" ".join(lint_options), toLocation(manifest))
-    cmd += "rm $CP_FILE; "
+    
+    #Delete tmp file and tmp.bak file
+    cmd += "rm $CP_FILE*; "
```

Comment:
This is needed because sed -i.bak is used above, and I left that alone for the sake of historical decisions on backing it up. I assume it was used to be preserved in the case of debugging failures.